### PR TITLE
docs(chips): use consistent data structure in autocomplete example

### PR DESCRIPTION
**Pull request by banminkyoz**

### CODE REVIEW

File: src/material-examples/chips-autocomplete/chips-autocomplete-example.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
       *ngFor="let fruit of fruits"
       [selectable]="selectable"
       [removable]="removable"
-      (remove)="remove(fruit)">
-      {{fruit.name}}
+      (removed)="remove(fruit)">
+        {{fruit}}
```
Comment (rafaelss95) [inline src/material-examples/chips-autocomplete/chips-autocomplete-example.html]: I think you've accidentally added extra spaces here.

Comment (banminkyoz) [reply src/material-examples/chips-autocomplete/chips-autocomplete-example.html]: haha. yes it was mine. do i need to remove them ? 😄

Comment (jelbourn) [reply src/material-examples/chips-autocomplete/chips-autocomplete-example.html]: Yes please

Comment (banminkyoz) [reply src/material-examples/chips-autocomplete/chips-autocomplete-example.html]: Sorry for too late. i'v fixed it ^_^

**Pull request merged**